docs/plugin_development.rst

```diff
@@ -352,48 +427,57 @@ After ``cookicutter`` finishes its job, you'll get the following tree structure
+     release              package and upload a release
+     dist                 builds source and wheel package
+     install              install the package to the active Python's site-packages
+     develop              install the package to the active Python's site-packages
 
+* ``setup.py``.
+  It helps you install, package and distribute your plugin with Setuptools, the standard for distributing Python packages.
+  In this file the ``entry_points`` parameter of ``setup`` is quite important, as it is what allows Spyder to identify this package as a plugin, and know how to access its functionality.
 
-The ``spyder-pomodoro-timer`` folder has the name you introduced when running ``cookiecutter``. Inside this you'll see a folder called ``spyder``, where we will place the code of our plugin.
+The ``spyder-pomodoro-timer`` folder has the same name you entered when you ran ``cookiecutter``.
+Inside it you will see a folder called ``spyder``, where we will place the code of our plugin.
 
 In the ``spyder`` directory you'll find the following files:
 
-* ``api.py``: where the functionality of the plugin is exposed to the rest of Spyder. That would allow additional functionality to be added from other plugins.
+* ``api.py``.
+  Exposes the functionality of the plugin to the rest of Spyder.
+  That allows adding additional functionality from other plugins.
 
-* ``plugin.py``: is the core of the plugin. Depending on the type of plugin we created, here you'll see an instance of ``SpyderDockablePlugin`` or ``SpyderPluginV2``.
+* ``plugin.py``.
+  This is the core of the plugin.
+  Depending on the type of plugin created, you will see an instance of either ``SpyderDockablePlugin`` or ``SpyderPluginV2``.
 
   * If it is a ``SpyderPluginV2`` you should set a constant class named ``CONTAINER_CLASS`` with an instance of ``PluginMainContainer``.
   * If it is a ``SpyderDockablePlugin`` you should set a constant class named ``WIDGET_CLASS`` with an instance of ``PluginMainWidget``.
 
-* ``container.py``: only used for ``SpyderPluginV2`` plugins. This file contains an instance of ``PluginMainContainer`` that holds a reference to all graphical elements (or widgets) that the plugin is going to add to the interface. This is necessary because Qt requires widgets to be children of other widgets before using them (otherwise they appear as floating windows). Since ``SpyderPluginV2`` is not a widget, we need a data structure (i.e. the container) that is a widget for that.
+* ``container.py``.
+  It is only used for ``SpyderPluginV2`` plugins.
+  This file contains an instance of ``PluginMainContainer``, which contains a reference to all graphical elements (or widgets) that the plugin will add to the interface.
+  This is necessary because Qt requires widgets to be children of other widgets before using them (otherwise they appear as floating windows).
+  Since ``SpyderPluginV2`` is not a widget, we need a data structure (i.e. the container) that is a widget for it.
```

Comment:
```suggestion
  Since ``SpyderPluginV2`` is not a widget, the container (which is a widget) serves this role for us.
```

I found the existing text very confusing, so after some parsing and thought I rewrote it like this